app/models/User.js

```diff
@@ -170,6 +170,10 @@ module.exports = (User = (function () {
     isSmokeTestUser () { return User.isSmokeTestUser(this.attributes) }
     isIndividualUser () { return !this.isStudent() && !User.isTeacher(this.attributes) }
 
+    getSubscriptionLevel () {
+      return (this.isPaidTeacher() || this.isPremium()) ? 'paid' : 'free'
```

Comment:
this is not a correct check. If u see comment in isPaidTeacher
```
// TODO: this doesn't actually check to see if they are paid (having prepaids), confusing
 // use isPaidTeacher from me.js in vuex store for vue
```
Use the check from vue store me.js or fetch prepaids and check